### DISCUSSION TITLE
Beatmap Listing: Sorting mapsets with the little difficulty colors. (with concept art !!)

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#18295] [Ideas] [answered]
So I've been using lazer primarily for a little, mainly because I stopped playing for pp and you could download maps in-game. It's been very fun for the most part, but I found that if I wanted 6-8 star maps to dick around with (that's what I play primarily), it would be kind of tedious to scroll for 2-3 minutes every time just to find a few maps that you'd find catchy and interesting to play.

I attached a mock-up of my idea to this thread, decided to go with the difficulty colors because I've realized you become awfully accustomed to them after a little. Like, associating a high 7 star with dark purple and then 9-10+ stars are just pure black was kind of a cool idea lmao.

_(Sidenote: my mock-up doesn't include all colors, but you should be able to get the idea)_
![mockup](https://user-images.githubusercontent.com/51036913/168496854-c934a2e0-ffb1-4eea-9b9c-ccf0aa804913.png)

**Comment by smoogipoo** [comment, marked as answer]
You should request this first at https://github.com/ppy/osu-web

**Reply by ghost** [reply]
gotcha

**Comment by bdach** [comment]
This is already doable if you input i.e. `stars>7` in the search box.
